higlass/__init__.py

```diff
@@ -6,6 +6,22 @@
 
 
 def _jupyter_nbextension_paths():
+    ViewConf(
```

Comment:
What is this doing? Was this accidentally added? If not, could you describe the change in the PR?

Reply:
Oh, this definitely looks like it was accidentally added. I'll remove it right now.

Reply:
I'm baffled as to how it got in there though 🤔